vispy/visuals/tests/test_image.py

```diff
@@ -263,4 +263,30 @@ def test_image_vertex_updates():
             build_vertex_mock.assert_called_once()
 
 
+@requires_application()
+def test_change_clim_float():
+    """
+    Test that with an image of floats, clim is correctly set from the first try
+
+    see https://github.com/vispy/vispy/pull/2245
+    """
```

Comment:
```suggestion
    """Test that with an image of floats, clim is correctly set from the first try.

    See https://github.com/vispy/vispy/pull/2245.
    """
```

I'm picky about docstrings.

Reply:
Hehe, I actually thought about this, and still failed :P